Gofer.NET.Utils/JsonSkipCancellationTokenConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading;
+
+using Newtonsoft.Json;
+
+namespace Gofer.NET.Utils
+{
+    public class JsonSkipCancellationTokenConverter : JsonConverter
```

Comment:
Unless you can provide a compelling reason why the cancellation token should be excluded from the serialization, this functionality should be removed altogether.

Reply:
I've added cancellation support for enqueued tasks. Json.Net cannot serialize CancellationToken anyway so it was an easiest fix I've could imagine.